src/commands/ls_remote.rs

```diff
@@ -1,24 +1,56 @@
 use crate::config::FnmConfig;
 use crate::remote_node_index;
+use crate::user_version::UserVersion;
+
+use colored::Colorize;
 use thiserror::Error;
 
 #[derive(clap::Parser, Debug)]
-pub struct LsRemote {}
+pub struct LsRemote {
+    /// Filter for versions
+    filter: Option<UserVersion>,
```

Comment:
Is `--filter` the option name you want to settle on?

Looking at the value enum it'd support, and the addition of `--lts` as an alternative filter (_semi-compliementary_), perhaps `--semver` is more appropriate?

https://github.com/Schniz/fnm/blob/3bb442d66d16c1f4e47045e64387fc5924b2dd1b/src/user_version.rs#L4-L10

Something like these?

```suggestion
    /// Filter output by semver
    filter: Option<UserVersion>,
```

```suggestion
    /// Filter versions by pattern
    semver: Option<UserVersion>,
```